### PR TITLE
[MXNET-284] Update release version text to 1.2.0

### PR DESCRIPTION
**Pull request by kpmurali**
## Description ##
In the home page for the site, the static text needs to be updated to the 1.2.0 version.

### Changes ###
- [x] Modified the corresponding text in the docs/_static/mxnet-theme/index.html file

### CODE REVIEW

**Comment by anirudh2290** [comment]
is this still wip ?

File: docs/_static/mxnet-theme/index.html [hunk 1]
```diff
@@ -21,21 +21,21 @@
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>Apache MXNet 1.1.0 Released</h3>
-        <p>We're excited to announce the release of MXNet 1.1.0! Check out the release notes for latest updates.</p>
-        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.1.0">Learn More</a>
+        <h3>MXNet 1.2.0.rc0 Released</h3>
+        <p>We're excited to announce the release of MXNet 1.2.0.rc0! Check out the release notes for latest updates.</p>
+        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.2.0">Learn More</a>
```
Comment (anirudh2290) [inline docs/_static/mxnet-theme/index.html]: The link should be : `https://github.com/apache/incubator-mxnet/releases/tag/1.2.0`

Comment (kpmurali) [reply docs/_static/mxnet-theme/index.html]: Ok, thanks for the feedback. I will change this.

**Comment by kpmurali** [comment]
I have updated the link to be : https://github.com/apache/incubator-mxnet/releases/tag/1.2.0.rc0.

**Comment by kpmurali** [comment]
This PR is no longer WIP.

File: docs/_static/mxnet-theme/index.html [hunk 2]
```diff
@@ -21,21 +21,21 @@
   <div class="container">
     <div class="row">
       <div class="col-lg-4 col-sm-12">
-        <h3>Apache MXNet 1.1.0 Released</h3>
-        <p>We're excited to announce the release of MXNet 1.1.0! Check out the release notes for latest updates.</p>
-        <a href="https://github.com/apache/incubator-mxnet/releases/tag/1.1.0">Learn More</a>
+        <h3>MXNet 1.2.0.rc0 Released</h3>
```
Comment (eric-haibin-lin) [inline docs/_static/mxnet-theme/index.html]: why announce rc0? I don't remember such announcements for previous releases

Comment (anirudh2290) [reply docs/_static/mxnet-theme/index.html]: This is a new change and we are adding a drop-down menu item for users to preview 1.2.0rc0 and that is the reason for the announcement.

**Comment by aaronmarkham** [comment]
This allows people to review the updates on the site and look at API docs, etc. We will follow up with another update to master when the RC is accepted and change it to be announcing 1.2.0.

**Pull request merged**